docs-src/concepts/what-is-a-search-attribute.md

```diff
@@ -29,23 +29,28 @@ A Temporal Cluster has a set of default Search Attributes already available.
+| StartTime                  | Datetime | The time at which the Workflow Execution started.                                                                                                                            |
+| CloseTime                  | Datetime | The time at which the Workflow Execution completed.                                                                                                                          |
+| ExecutionTime              | Datetime | Same as StartTime for the most cases but different for cron Workflows and retried Workflows. For them it is the time at which the Workflow Execution actually begin running. |
+| RunId                      | Keyword  | Identifies the current Workflow Execution Run.                                                                                                                               |
+| ExecutionDuration          | Int      | The time needed to run the Workflow Execution. Available only for closed Workflows.                                                                                          |
+| HistoryLength              | Int      | The number of events in the history of Workflow Execution. Available only for closed Workflows.                                                                              |
+| StateTransitionCount       | Int      | The number of times that Workflow Execution has persisted its state. Available only for closed Workflows.                                                                    |
+| TaskQueue                  | Keyword  | Task Queue used by Workflow Execution.                                                                                                                                       |
+| TemporalChangeVersion      | Keyword  | If Workflow versioning is enabled, list of change/version pairs will be stored here.                                                                                         |
+| BinaryChecksums            | Keyword  | List of binary Ids of Workers that run the Workflow Execution.                                                                                                               |
+| BatcherNamespace           | Keyword  | Used by internal batcher to indicate the Namespace where batch operation was applied to.                                                                                     |
+| BatcherUser                | Keyword  | Used by internal batcher to indicate the user who started the batch operation.                                                                                               |
+| NamespaceId                | Keyword  | Identifies the Namespace of the Search Attribute.                                                                                                                            |
```

Comment:
so NamespaceId is an attribute on the ES index, but I don't think it is considered a registered search attribute because all searches are by definition within the context of a single namespace. Check with @rodrigozhou or @alexshtin on this one though

Reply:
Yes, this one is kinda hidden, can't be used directly and shouldn't be mentioned here.